seatunnel-core/seatunnel-flink-starter/src/main/java/org/apache/seatunnel/core/starter/flink/execution/AbstractPluginExecuteProcessor.java

```diff
@@ -50,15 +50,15 @@
     protected AbstractPluginExecuteProcessor(List<URL> jarPaths, List<? extends Config> pluginConfigs, JobContext jobContext) {
         this.pluginConfigs = pluginConfigs;
         this.jobContext = jobContext;
-        this.plugins = initializePlugins(jarPaths, pluginConfigs);
+        this.plugins = initializePlugins(jarPaths, pluginConfigs, jobContext);
```

Comment:
I think we can use jobContext directly in initializePlugins without adding parameters.

Reply:
But initializePlugins's config will only include the specified(transform,souce,sink) config;

Reply:
ok, got it ,thanks